packages/error-utils/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', { targets: { node: 'current' } }],
+    '@babel/preset-typescript',
+  ],
+};
```

Comment:
Lets remove babel. We use swc. 

In fact, we don't need it at all because TS has `target`

Reply:
Jest errored without this and told me to add it. Ill look how other packages are doing it and fix

Reply:
Probably needs a jest config file like this
https://github.com/vercel/vercel/blob/51fe09d5e96df72877e5daf21929aa7ec43cd382/packages/fs-detectors/jest.config.js#L5-L6